odoo/models.py

```diff
@@ -1699,15 +1699,21 @@ def _read_group_process_groupby(self, gb, query):
                 'day': 'dd MMM yyyy', # yyyy = normal year
                 'week': "'W'w YYYY",  # w YYYY = ISO week-year
                 'month': 'MMMM yyyy',
+                'bimester': 'MMM yyyy',
                 'quarter': 'QQQ yyyy',
+                'trimester': 'MMM yyyy',
+                'semester': 'MMM yyyy',
                 'year': 'yyyy',
             }
             time_intervals = {
                 'hour': dateutil.relativedelta.relativedelta(hours=1),
                 'day': dateutil.relativedelta.relativedelta(days=1),
                 'week': datetime.timedelta(days=7),
                 'month': dateutil.relativedelta.relativedelta(months=1),
+                'bimester': dateutil.relativedelta.relativedelta(months=2),
                 'quarter': dateutil.relativedelta.relativedelta(months=3),
+                'trimester': dateutil.relativedelta.relativedelta(months=4),
```

Comment:
trimester are 3 months, equivalent to quarter.

quadmester or quadrimester are used for 4 months.

Reply:
It's ambivalent (https://en.wikipedia.org/wiki/Trimester), but accounting context favors your suggestion.

Reply:
But where do you see the ambivalence? In neither of those cases in the wiki the trimester has 4 months.

Reply:
In fact, I think it would be more correct to have Trimester as the "neutral" three months's name, and put "Quarter" in the l10n_us translation 😏

Reply:
I think I overlooked that students have a looot of holidays :smile: